doc/bash-completion.sh

```diff
@@ -691,6 +691,7 @@ _restic_copy()
     flags_with_completion=()
     flags_completion=()
 
+    two_word_flags+=("--copy-workers")
```

Comment:
The doc changes are generated automatically when releasing a new restic version. Please remove them from the PR.